using-spring-data-jpa/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+quarkus.datasource.url=jdbc:postgresql:quarkus_test
```

Comment:
Apparently, people would prefer we specify the host: https://github.com/quarkusio/quarkus-quickstarts/pull/260/files#diff-284b129d02c2070784925ee9ac16a157R1

Reply:
Interesting, I'll update then :)